drivers/modem/hl7800.c

```diff
@@ -1697,8 +1719,16 @@ static bool on_cmd_atcmdinfo_operator_status(struct net_buf **buf, uint16_t len)
 
 	out_len = net_buf_linearize(value, sizeof(value), *buf, 0, len);
 	value[out_len] = 0;
-	LOG_INF("Operator: %s", log_strdup(value));
 
+	/* For AT+COPS=?, result is most likely longer than size of log string */
```

Comment:
Is this change related, it's not reflected in the commit message? Looks more like some bugfix to me.

Reply:
new PR here: https://github.com/zephyrproject-rtos/zephyr/pull/37849